crates/core/component/dex/src/component/tests.rs

```diff
@@ -866,6 +866,9 @@ async fn basic_cycle_arb() -> anyhow::Result<()> {
 /// The issue was that we did not treat the spill price as a strict
 /// upper bound, which is necessary to ensure that the arbitrage logic
 /// terminates.
+///
+/// This test also ensures that the created `SwapExecution` has the
+/// correct data.
```

Comment:
```suggestion
///
/// *Arbitrage execution record bug:*
/// This test also ensures that the created `SwapExecution` has the
 /// correct data. (See #3790).
```

Good idea to test this but can we add context that it's separate from the testnet 53 issue